Panic on specific logger log level

`kubectl istiolog alpine -n default -l debug -f` works 
but below doesn't

```shell
$ kubectl istiolog alpine -n default -l http:debug -f
panic: unrecognized logger name: http

goroutine 1 [running]:
github.com/TejaBeta/kubectl-istiolog/cmd.glob..func1(0x318e680?, {0xc0009918c0, 0x1, 0x6?})
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/cmd/root.go:49 +0x10c
github.com/spf13/cobra.(*Command).execute(0x318e680, {0xc00013a010, 0x6, 0x6})
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/vendor/github.com/spf13/cobra/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x318e680)
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/vendor/github.com/spf13/cobra/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/vendor/github.com/spf13/cobra/command.go:902
github.com/TejaBeta/kubectl-istiolog/cmd.Execute()
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/cmd/root.go:55 +0x25
main.main()
        /home/runner/work/kubectl-istiolog/kubectl-istiolog/main.go:28 +0x4e
```

I'll submit a PR

--- a/internal/kubectl-istiolog.go
+++ b/internal/kubectl-istiolog.go
@@ -178,14 +178,14 @@ func handleLog(logLevel string, pod string, namespace string) error {
 				return fmt.Errorf("unrecognized logging level: %v", ol)
 			}
 		} else {
-			invalidLogName := false
+			invalidLogName := true
 			loggerLevel := regexp.MustCompile(`[:=]`).Split(ol, 2)
 
 			for _, logName := range allLoggers {
 				if logName == loggerLevel[0] {
+					invalidLogName = false
 					break
 				}
-				invalidLogName = true
 			}
 
 			if invalidLogName {

--- a/internal/kubectl-istiolog_test.go
+++ b/internal/kubectl-istiolog_test.go
@@ -14,6 +14,7 @@ package internal
 
 import (
 	"context"
+	"regexp"
 	"testing"
 
 	appv1 "k8s.io/api/core/v1"
@@ -115,6 +116,33 @@ func TestIstioLoggerName_A001(t *testing.T) {
 	}
 }
 
+// We successfully parsed logger name & level and we failed on envoy call
+func successfullyParsedLoggerNameAndLevel(err error) bool {
+	errString := err.Error()
+	return err != nil && regexp.MustCompile("failed to execute command on Envoy").MatchString(errString)
+}
+
+func TestIstioLoggerName_A002(t *testing.T) {
+	cs := testclient.NewSimpleClientset()
+	input := &appv1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "unit-test-pod"}}
+
+	options := options{
+		clientset: cs,
+		namespace: "unit-test-namespace",
+	}
+
+	_, err := cs.CoreV1().Pods(options.namespace).Create(context.TODO(), input, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	err = options.KubectlIstioLog("unit-test-pod", "debug", false)
+
+	if !successfullyParsedLoggerNameAndLevel(err) {
+		t.Errorf("Error while using illegal loggerLevel")
+	}
+}
+
 func TestIstioLoggerLevel_A001(t *testing.T) {
 	cs := testclient.NewSimpleClientset()
 	input := &appv1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "unit-test-pod"}}
@@ -132,5 +160,25 @@ func TestIstioLoggerLevel_A001(t *testing.T) {
 	err = options.KubectlIstioLog("unit-test-pod", "debug:hello", false)
 	if err == nil {
 		t.Errorf("Error while using illegal loggerLevel")
+	}
+}
+
+func TestIstioLoggerLevel_A002(t *testing.T) {
+	cs := testclient.NewSimpleClientset()
+	input := &appv1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "unit-test-pod"}}
+
+	options := options{
+		clientset: cs,
+		namespace: "unit-test-namespace",
+	}
+
+	_, err := cs.CoreV1().Pods(options.namespace).Create(context.TODO(), input, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	err = options.KubectlIstioLog("unit-test-pod", "http:debug", false)
+	if !successfullyParsedLoggerNameAndLevel(err) {
+		t.Errorf("Error while using illegal loggerName")
 	}
 }

